BeagleBone_black/src/sensors/main.cpp

```diff
@@ -55,12 +57,18 @@ Main::Main(uint8_t id, Logger& log)
       battery_init_(false)
 {
   // @TODO (Ragnor) Add second Keyence?
+  if (sys_.fake_sensors || sys_.fake_keyence) {
+    fake_keyence_ = new FakeGpioCounter(log, "../BeagleBone_black/data/in/fake_keyence_input.txt");
```

Comment:
It would be better to have the same variable for both fake and normal keyence. That way, we don't need more `if`s everywhere we use keyence. Also the whole idea of a fake sensor is that anyone using it shouldn't be able to tell if it's fake or not (perhaps except for differences in constructor).